torch/_torch_docs.py

```diff
@@ -4350,13 +4352,6 @@ def merge_dicts(*dicts):
 :attr:`dim`. And ``indices`` is the index location of each maximum value found
 (argmax).
 
-.. warning::
```

Comment:
The note actually needs to go with this version of max, which returns indices, not the first version which does not.

Further, the note should be slightly expanded, "If there are multiple maximal values in a reduced row then the indices of the first maximal value is returned." 

Parallel comments for the note below, too.